packages/rich-text/src/plugins/Paragraph/index.tsx

```diff
@@ -33,7 +33,7 @@ export function createParagraphPlugin(): PlatePlugin {
     onKeyDown: getToggleElementOnKeyDown(BLOCKS.PARAGRAPH),
     deserialize: deserializeElement(BLOCKS.PARAGRAPH, [
       {
-        nodeNames: ['P'],
+        nodeNames: ['P', 'DIV'],
```

Comment:
We need to be careful when adding `DIV` here, sometimes div is a container with no text at all, and this will result in a lot of empty paragraphs for each div.

Look at my first try of adding `div` as a valid node: https://github.com/contentful/field-editors/blame/4efe00650c34261f90bbd73e5fa430f34e3ab438/packages/rich-text/src/plugins/Paragraph/index.tsx#L36-L56

Did you try copying and pasting embedded blocks with this approach?

Reply:
It doesn't work when copying embedded blocks because they contain div, and according to this logic, divs should be considered as paragraphs, which is not true in some cases: https://recordit.co/MW6qrjCRn6

Reply:
When copying some texts that contain a lot of containers as div, each div will result in an empty paragraph as I commented previously:

![image](https://user-images.githubusercontent.com/954889/140962708-8021b544-b9dc-421b-b329-972febc88101.png)

Imagine this scenario:

`div > div > div > section > div > p + img + p`

Each div here will be considered as a paragraph. I'm not saying this is wrong, it might be acceptable but it's worth discussing this with Malin maybe?

Reply:
@cezarsmpio thanks for the explanation. I overlooked that as I was focused at getting a div into a paragraph.

I came up with a new version looking at yours, just not as "plain". So, your previous attempt was missing the nesting control, as mine now, right? Do we need the check for emptiness?

![Kapture 2021-11-10 at 08 36 07](https://user-images.githubusercontent.com/71259950/141073284-4b82d202-c728-4aa8-8863-b247562761f3.gif)

Reply:
It looks much better! :) 

> Do we need the check for emptiness?

I think we should... Editors won't need to keep deleting those meaningless empty paragraphs. What do you think?

Reply:
A case with some nested html:
![Kapture 2021-11-10 at 09 13 09](https://user-images.githubusercontent.com/71259950/141078043-26d57b6a-c77c-4d44-890e-4afdbe0c35ab.gif)